wandb/sdk/launch/sweeps/scheduler_sweep.py

```diff
@@ -38,120 +37,114 @@ class SweepScheduler(Scheduler):
     def __init__(
         self,
         *args: Any,
-        sweep_id: Optional[str] = None,
         num_workers: int = 4,
-        heartbeat_thread_sleep: float = 0.5,
-        heartbeat_queue_timeout: float = 1,
-        main_thread_sleep: float = 1,
+        worker_sleep: float = 0.1,
```

Comment:
These are very short sweeps, just so I can be educated, why are we reducing them so?

Reply:
Good catch, I had them short for testing purposes. 1 seconds seems like a better default.